### ISSUE TITLE
Can't get it to work (scale is not a function)

### ISSUE THREAD

**Issue by av01d** [elrumordelaluz/scale-that-svg#2]
Your documentation is very sparse... so sparse, that I can't get the simplest example to work.
I successfully installed (via yarn) the package, but can't get it to run. I'm sure it's my mistake, but I don't know where to look.
Here's my sample code (test.js):
```
const fs = require('fs');
const scale = require('scale-that-svg');

fs.readFile('./test.svg', (err, input) => {
  scale(input, { scale: 0.5 }).then(scaled => {
    console.log({ scaled })
  })
})
```
Running `node test.js` results in 
```
TypeError: scale is not a function
    at fs.readFile (/var/www/tmp/svgResize/test.js:5:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:53:3)
```
I'm running node (v10.21.0) on Debian.

**Comment by elrumordelaluz** [comment]
Hi @av01d, thanks for open the _Issue_.

Should be fixed in `scale-that-svg@1.0.4` importing like:

```js
const { scale } = require('scale-that-svg');
```

Closing, but feel fre to re-open in case you need!

Feel free also to make PRs to any improve you think could be done in the _docs_. Thansk!

**Comment by av01d** [comment]
Thanks, it works great now!
